### ISSUE TITLE
`bibtex_file_path` setting is ignored

### ISSUE THREAD

**Issue by igelstorm** [mangecoeur/Citer#44]
* The setup instructions in the README say that the BibTeX file should be configured in `bibtex_file_path` in `Citer.sublime-settings`.
* When I set the `bibtex_file_path` setting to an existing, valid BibTeX file, the file isn't found and I can't insert any citations
* When I set the `bibtex_file` setting to the same BibTeX file, the plugin works as expected, and I can insert citations

## Expected behaviour

* Either setting `bibtex_file_path` should be sufficient for the plugin to work
* Or the documentation should be changed to clarify that `bibtex_file` needs to be set

## Background

There's some discussion about this in issue https://github.com/mangecoeur/Citer/issues/41 and PR https://github.com/mangecoeur/Citer/pull/42. It looks like this is a regression that was introduced by accident in PR https://github.com/mangecoeur/Citer/pull/42?

**Comment by choc96208** [comment]
I found this too. On a whim I changed it to `bibtex_file` from the [citebibtex](https://github.com/sjpfenninger/citebibtex) documentation and it worked! Lots of people will just copy the template in the readme.md and not know about this change needed.

**Comment by frederic-santos** [comment]
> Lots of people will just copy the template in the readme.md and not know about this change needed.

Yep, same problem here. At this stage, the `README.md` is clearly misleading. I can do a PR if needed, but the docs need an update.
